plugins/analytics/public/dockerhub.py

```diff
@@ -77,40 +80,11 @@ def image_full_details(image):
         return image_metadata
 
 
-class DockerImageInspect(task.OneShotTask):
-    """DockerImageInspect analytics queries docker hub to get more
-    details related to docker_image observable.
-
-    This analytics adds several information as context to docker_image:
-    * Image stats: provide details about pulls, registered, updated
-    * Image details: provide information related to image tags including layers information and instructions
-    * User details: provides details about the user owning the image.
-
-    This analytics also creates new observables:
-    * generic observable with tag type:user_account to link to a user
-    * sha256 including image's digest, layers' digests and added files' digests
-    """
-
-    _defaults = {
-        "name": "DockerImageMetadata",
-        "description": "Fetch metadata from docker hub for a docker image",
-    }
-
-    acts_on: list[ObservableType] = [
-        ObservableType.docker_image,
-    ]
-
+class DockerHubObservables:
```

Comment:
Do we really need a class here? None of the methods defined in this class are actually using `self`, could they be defined as module-level methods?

Reply:
makes sense!

Reply:
refactored in 428cfb0102bf35123965a5608e78aed7fb696978